jsk_arc2017_baxter/euslisp/lib/pick-interface.l

```diff
@@ -130,6 +130,7 @@
     ;; TODO: in-hand verification
     ;; TODO:   target-obj = in-hand object  -> :set_target_cardboard
     ;; TODO:   target-obj != in-hand object -> :return_object
+    (setq target-obj t)
```

Comment:
> I think smach doesn't die because of it.

At least, 
```
$ roslaunch jsk_arc2017_baxter baxter.launch moveit:=true pick:=true
$ roslaunch jsk_arc2017_baxter setup_for_pick.launch
$ roslaunch jsk_arc2017_baxter pick.launch json_dir:=$(rospack find jsk_arc2017_common)/data/json/easy_pick_task moveit:=true
```
dies due to smach error without this change.
Currently, I can't write simple smach test due to lack of knowledge, so can you check?